File: src/main/java/com/akagiyui/drive/model/StorageFile.java
package com.akagiyui.drive.model;

import lombok.Data;
import lombok.experimental.Accessors;

/**
 * 存储中的文件
 *
 * @author AkagiYui
 */
@Data
@Accessors(chain = true)
public class StorageFile {
    /**
     * 文件名
     */
    private String key;

    /**
     * 文件大小
     */
    private long size;

    /**
     * 文件类型
     */
    private String type;

    /**
     * 文件哈希
     */
    private String hash;
}

File: app/src/main/kotlin/com/akagiyui/drive/model/StorageFile.kt
package com.akagiyui.drive.model

/**
 * 存储中的文件
 *
 * @author AkagiYui
 */
class StorageFile(
    /**
     * 文件名
     */
    val key: String,
    /**
     * 文件大小
     */
    val size: Long,
    /**
     * 文件类型
     */
    val type: String,
    /**
     * 文件哈希
     */
    val hash: String,
)
